app/scripts/modules/netflix/fastProperties/domain/propertyCommnadType.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum PropertyCommandType {
```

Comment:
Do we send these values to the server or are they only ever used on the client side?  One tricky thing about TS enums is that they rely on the values being stored as ordinals (and not strings) in one's datastore.  I.e. `PropertyCommandType['CREATE'] === 0` and not `PropertyCommandType.CREATE`.